src/sentry/static/sentry/app/views/settings/components/dataPrivacyRules/dataPrivacyRulesForm/dataPrivacyRulesFormEventIdStatusIcon.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import ControlState from 'app/views/settings/components/forms/field/controlState';
+import {t} from 'app/locale';
+import Tooltip from 'app/components/tooltip';
+import {IconClose, IconCheckmark} from 'app/icons';
+
+import {EventIdStatus} from './types';
+
+type Props = {
+  onClickIconClose: () => void;
+  status?: EventIdStatus;
+};
+
+const DataPrivacyRulesFormEventIdStatusIcon = ({status, onClickIconClose}: Props) => {
+  switch (status) {
+    case EventIdStatus.ERROR:
+    case EventIdStatus.INVALID:
+    case EventIdStatus.NOT_FOUND:
+      return (
+        <CloseIcon onClick={onClickIconClose}>
+          <Tooltip title={t('Clear Event ID')}>
+            <IconClose color="red" />
+          </Tooltip>
+        </CloseIcon>
+      );
+    case EventIdStatus.LOADING:
+      return <ControlState isSaving />;
+    case EventIdStatus.LOADED:
+      return <IconCheckmark color="green" />;
+    default:
+      return null;
+  }
+};
+
+export default DataPrivacyRulesFormEventIdStatusIcon;
+
+const CloseIcon = styled('div')`
+  cursor: pointer;
+  :first-child {
```

Comment:
Is this selector for tooltip?

Reply:
yes.. I cannot style the tooltip using `styled(Tooltip)`

Reply:
Should we use `popperStyle` prop of tooltip?

Reply:
I would prefer not use inline styles

Reply:
IMO inline styles here would make it more clear of what you are styling. `first-child` selector can also be brittle (e.g. what if you add something inside the wrapper?)